### ISSUE TITLE
Undefined method

### ISSUE THREAD

**Issue by mvdnbrk** [edgarjs/alfred-github-repos#24]
```
`search_repos': undefined method `filter' for #<Array> (NoMethodError) from bin/github-repos:72:in `search'
```

**Comment by gabyquiles** [comment]
I had the same problem, and fixed it by going into the workflow folder, then edit the file `lib/github/api.rb`. Once you have that file opened edit line 38 and change from 
`repos_filtered = repos.filter do |i|`
to 
`repos_filtered = repos.select do |i|`
I'm hesitant to open a pull request since it seems that filter is the new way of filtering arrays. I hope this helps.

https://apidock.com/ruby/Array/filter

**Comment by edgarjs** [comment]
This is fixed in new version 2.1.3
